lib/tasks/mastodon.rake

```diff
@@ -124,7 +124,7 @@ namespace :mastodon do
           host: env['REDIS_HOST'],
           port: env['REDIS_PORT'],
           password: env['REDIS_PASSWORD'],
-          driver: :hiredis,
+          ssl_params: { verify_mode: OpenSSL::SSL::VERIFY_NONE },
```

Comment:
Won't this enforce SSL/TLS? We might need a question for this because not all Redis hosts will be set up with SSL (even self-signed).

Reply:
Thanks for pointing this out! It looks like redis-rb connects through TLS with `ssl: true`, i.e.,:

```ruby
redis_options = {
  host: env['REDIS_HOST'],
  port: env['REDIS_PORT'],
  password: env['REDIS_PASSWORD'],
  ssl: true,
  ssl_params: { verify_mode: OpenSSL::SSL::VERIFY_NONE },
}
```

Without `ssl: true`, the `ssl_params` is ignored.

Reply:
Sounds like we probably need a prompt to ask about SSL for Redis then.